<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorControlForm</class>
 <widget class="QWidget" name="DetectorControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>81</y>
     <width>60</width>
     <height>79</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="initialiseConnection">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialise Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abortDAQ">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abort DAQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="image2Label">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>81</y>
     <width>20</width>
     <height>79</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="quadrantLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Quadrant</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="image3Label">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>81</y>
     <width>20</width>
     <height>79</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="quadrant">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="detectorTemperature">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>12</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Idle</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>80</y>
     <width>61</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel2">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>61</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Combined Scale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageCountLabel">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>80</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Images</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="imageCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel1">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>61</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string> Separate Scale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>159</y>
     <width>61</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>80</y>
     <width>1</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="collectImages">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Collect Images</string>
   </property>
  </widget>
  <widget class="QLabel" name="image1Label">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>81</y>
     <width>20</width>
     <height>79</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dataAcquisitionModeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="terminateConnection">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>11</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminate Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reconnect">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>11</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-Connect</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>1</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="housingTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Housing Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="housingTemperature">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="housingHumidityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Housing Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="housingHumidity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="fingerTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Finger Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="fingerTemperature">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="dewPointLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dew Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="dewPoint">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="fingerTemperatureLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Detector Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="dewPointLabel_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>12</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="setFingerTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Finger Temperature</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logFileTimestamp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>356</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Append Timestamp?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logFileDirectoryButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>328</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="logFilePrefixLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>356</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Log File Prefix</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="logFilePrefix">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>356</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="logFileDirectory">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logFileDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Log File Directory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="biasCurrent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="biasLastRefreshedLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias last refreshed at</string>
   </property>
  </widget>
  <widget class="QLabel" name="biasCurrentLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="biasLastRefreshed">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="biasVoltageButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias Voltage?</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="loggingEnabled">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging?</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="setFingerTemperature">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-273.100000000000023</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="setFingerTemperatureButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
